src/components/card/Card.js

```diff
@@ -1,11 +1,25 @@
 import './Card.css'
 import deceased from '../images/deceased.png'
+import arrow from '../images/arrow-u-left-top.svg'
+import { useState } from 'react'
 
 function Card(props) {
+  const [isPageTurned, setIsPageTurned] = useState(false)
+  const [showEmoji, setShowEmoji] = useState('')
   const dead = props.alive
```

Comment:
This is not really best practice. The `props.alive` gives you `true` if the person is _alive_. You are saving it in a variable called `dead` - which makes me think that in this variable it is stored that the person is dead. Which is not true. I would name the variable: `isAlive`.